sql/sql_alter_instance.cc

```diff
@@ -269,6 +269,12 @@ bool Innodb_redo_log::execute() {
     return true;
   }
 
+  if (m_thd->backup_tables_lock.abort_if_acquired() ||
+      m_thd->backup_tables_lock.acquire_protection(
+          m_thd, MDL_STATEMENT, m_thd->variables.lock_wait_timeout)) {
```

Comment:
why MDL_STATEMENT? shouldn't it be MDL_TRANSACTION?

Reply:
Statement seemed to be enough, but now that I've checked it, the other backup lock is also acquired with MDL_TRANSACTION. Changed.

Reply:
thanks